raiden/__main__.py

```diff
@@ -5,7 +5,7 @@ def main():
     import gevent.monkey
     gevent.monkey.patch_all()
     from raiden.ui.cli import run
-    run()
+    run(auto_envvar_prefix='RAIDEN')
```

Comment:
Since not many of us are familiar with the click framework can you add a comment with a link to the explanation of [what `auto_envvar_prefix` does](http://click.pocoo.org/5/options/#values-from-environment-variables)?